interfaces/cython/cantera/test/test_kinetics.py

```diff
@@ -68,38 +68,39 @@ def test_legacy_reaction_rate(self):
         ct.make_deprecation_warnings_fatal() # re-enable fatal deprecation warnings
 
         fwd_rates = self.phase.forward_rate_constants
-        ix_3b = np.array([self.phase.reaction_type_str(i) == "three-body"
+        ix_3b = np.array([self.phase.reaction(i).reaction_type == "three-body"
             for i in range(self.phase.n_reactions)])
```

Comment:
Could this be
```suggestion
        ix_3b = np.array([r.reaction_type == "three-body" for r in self.phase.reactions])
```

Reply:
Yes, although `reactions` is a method, not a property. Imho, both
```
>>> gas.reactions()
>>> gas.species()
```
should probably be replaced by more Pythonic properties, i.e.
```
>>> gas.reactions
>>> gas.species
```
but it's somewhat disruptive and I don't want to add this to this PR.

Reply:
No problem, I think it can remain a method since you can also pass the index to it.